app/test/unit/git/cherry-pick-test.ts

```diff
@@ -494,23 +530,23 @@ describe('git/cherry-pick', () => {
       progress = []
     })
 
-    it('errors when given invalid revision range', async () => {
+    it('errors when given no commits', async () => {
       const progress = new Array<ICherryPickProgress>()
-      result = await cherryPick(repository, 'INVALID REF', p =>
-        progress.push(p)
-      )
+      result = await cherryPick(repository, [], p => progress.push(p))
       expect(result).toBe(CherryPickResult.UnableToStart)
     })
```

Comment:
Isn't this the same as https://github.com/desktop/desktop/pull/11976/files#diff-ce9ae9235d8e18f9cc6b0b7868c17378800f12f5cacf604cff2eb0e4354e833fR194 ? I think this specific test case made sense with the previous implementation, because `UnableToStart` was only returned when `cherryPick` had a progress callback, but I think we can remove this now, no? 🤔

Reply:
Yeah. Makes sense